### ISSUE TITLE
[aws-crt-cpp] Build error

### ISSUE THREAD

**Issue by ferben** [microsoft/vcpkg#27810]
Package: aws-crt-cpp[core]:arm64-osx -> 0.15.1#3

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.0.14000029
-    vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
    vcpkg-scripts version: 6f7ffeb18 2022-11-11 (3 days ago)

**To Reproduce**

`vcpkg install aws-sdk-cpp[s3] aws-sdk-cpp[s3]:arm64-osx-dynamic`

**Failure logs**

```
-- Downloading https://github.com/awslabs/aws-crt-cpp/archive/9ef58ff20df19e613c91c5f761e381c763da6810.tar.gz -> awslabs-aws-crt-cpp-9ef58ff20df19e613c91c5f761e381c763da6810.tar.gz...
-- Extracting source /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/downloads/awslabs-aws-crt-cpp-9ef58ff20df19e613c91c5f761e381c763da6810.tar.gz
-- Applying patch fix-cmake-target-path.patch
-- Applying patch fix-ios-build.patch
-- Using source at /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean
-- Configuring arm64-osx-dbg
-- Configuring arm64-osx-rel
CMake Warning at installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:311 (message):
  The following variables are not used in CMakeLists.txt:

      STATIC_CRT

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/aws-crt-cpp/portfile.cmake:13 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:131 (message):
    Command failed: /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/downloads/tools/cmake-3.24.0-osx/cmake-3.24.0-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/arm64-osx-dbg
    See logs for more information:
      /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/aws-crt-cpp/portfile.cmake:23 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/install-arm64-osx-dbg-out.log</summary>

```
[1/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/StringUtils.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/StringUtils.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/StringUtils.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/StringUtils.cpp
[2/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/DateTime.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/DateTime.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/DateTime.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/DateTime.cpp
[3/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/Types.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/Types.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/Types.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/Types.cpp
[4/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/crypto/Hash.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/crypto/Hash.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/crypto/Hash.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/crypto/Hash.cpp
...
Skipped 20 lines
...

/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/stdio.h:188:1: note: 'sprintf' has been explicitly marked deprecated here
__deprecated_msg("This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead.")
^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/sys/cdefs.h:215:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/external/cJSON.cpp:589:18: error: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Werror,-Wdeprecated-declarations]
        length = sprintf((char*)number_buffer, "%1.15g", d);
                 ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/stdio.h:188:1: note: 'sprintf' has been explicitly marked deprecated here
__deprecated_msg("This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead.")
^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/sys/cdefs.h:215:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/external/cJSON.cpp:595:22: error: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Werror,-Wdeprecated-declarations]
            length = sprintf((char*)number_buffer, "%1.17g", d);
                     ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/stdio.h:188:1: note: 'sprintf' has been explicitly marked deprecated here
__deprecated_msg("This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead.")
^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/sys/cdefs.h:215:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/external/cJSON.cpp:1028:21: error: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Werror,-Wdeprecated-declarations]
                    sprintf((char*)output_pointer, "u%04x", *input_pointer);
                    ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/stdio.h:188:1: note: 'sprintf' has been explicitly marked deprecated here
__deprecated_msg("This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead.")
^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/sys/cdefs.h:215:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
5 errors generated.
[11/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/ImdsClient.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/ImdsClient.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/ImdsClient.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/ImdsClient.cpp
[12/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/io/EventLoopGroup.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/io/EventLoopGroup.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/io/EventLoopGroup.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/io/EventLoopGroup.cpp
[13/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/UUID.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/UUID.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/UUID.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/UUID.cpp
[14/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/http/HttpRequestResponse.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/http/HttpRequestResponse.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/http/HttpRequestResponse.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/http/HttpRequestResponse.cpp
[15/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/io/ChannelHandler.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/io/ChannelHandler.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/io/ChannelHandler.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/io/ChannelHandler.cpp
[16/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/io/Bootstrap.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/io/Bootstrap.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/io/Bootstrap.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/io/Bootstrap.cpp
[17/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/io/HostResolver.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/io/HostResolver.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/io/HostResolver.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/io/HostResolver.cpp
[18/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/http/HttpProxyStrategy.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/http/HttpProxyStrategy.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/http/HttpProxyStrategy.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/http/HttpProxyStrategy.cpp
[19/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnectionManager.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnectionManager.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnectionManager.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/http/HttpConnectionManager.cpp
[20/28] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DAWS_MQTT_WITH_WEBSOCKETS -DAWS_USE_KQUEUE -DCJSON_HIDE_SYMBOLS -DDEBUG_BUILD -I/Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/include -isystem /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk -Wall -Wno-long-long -pedantic -Werror -std=gnu++11 -MD -MT CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnection.cpp.o -MF CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnection.cpp.o.d -o CMakeFiles/aws-crt-cpp.dir/source/http/HttpConnection.cpp.o -c /Volumes/Data/Designer.gitrepo/Src/AWS/vcpkg/buildtrees/aws-crt-cpp/src/c763da6810-766cf92b91.clean/source/http/HttpConnection.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

**Comment by LilyWangLL** [comment]
This issue will fixed by the upstream PR: https://github.com/awslabs/aws-crt-cpp/pull/431

**Comment by mads-modl-ai** [comment]
@LilyWangLL it looks like the upstream PR has been merged and tagged as version `0.18.16`, however, it doesn't seem like the ports in vcpkg supports versions above `0.15.1`.
Is there a way to update that? or would I have to get that version manualy?
